src/code/z_shrink_window.c

```diff
@@ -4,75 +4,76 @@
  */
 #include "prevent_bss_reordering.h"
 #include "global.h"
+#include "z64shrink_window.h"
 
 typedef struct {
     /* 0x0 */ s8 letterboxTarget;
     /* 0x1 */ s8 letterboxSize;
     /* 0x2 */ s8 pillarboxTarget;
     /* 0x3 */ s8 pillarboxSize;
-} ShrinkWindowContext; // size = 0x4
+} ShrinkWindow; // size = 0x4
```

Comment:
I wonder if it would be worth moving this struct to the new shrink_window header

Reply:
I think this is suppose to be internalized, that's why they have setters and getters for each member of this struct. So I think it's fine internally